integ-tests/src/main/java/com/google/base/TestBase.java

```diff
@@ -62,18 +62,30 @@ public static void assertListsEqual(
       Assert.fail(
           format(
               "DB view and Avro file have mutually exclusive row(s)"
-                  + lineSeparator() + "DB view '%s' has %d different row(s): %s"
-                  + lineSeparator() + "Avro file %s has %d different row(s): %s",
-              sqlPath, dbList.size(), dbListOutput, avroFilePath, avroList.size(), avroListOutput));
+                  + lineSeparator()
```

Comment:
Within format strings you can use `%n` instead of lineSeparator.

Reply:
Done.